api/NavigateEvent.json

```diff
@@ -118,7 +118,7 @@
           "support": {
             "chrome": {
               "version_added": "102",
-              "version_removed": "108"
+              "notes": "<code>canTransition</code> property become an alias of <code>canIntercept</code> property in Chrome 105."
```

Comment:
Yes, since `canIntercept` is on the standard track, let's move this over there, and remove this feature.